assets/_locales/en/messages.json

```diff
@@ -3668,6 +3668,18 @@
     "message": "1) Go to https://ao.arweave.net/#/delegate/ <br/> 2) Connect your Wander wallet <br/> 3) Find Wander in the project list <br/> 4) Click the Add button <br/> 5) Set your delegating % <br/> 6) Click the Confirm Delegation Preferences button <br/>  <br/> Done!",
     "description": "Announcement 3 body"
   },
+  "announcement_4_title": {
+    "message": "USDA Beta Access",
+    "description": "Announcement 4 title"
+  },
+  "announcement_4_description": {
+    "message": "Instant USDA Beta access for Reserve, Edge & Prime tier members",
```

Comment:
I added this to show as the short description on the Feed and wasn't included on the ticket or should we remove the description for all the announcements and use body when showing on the feed ? Or, Should we update it to new message ?

Reply:
Let's ask Matt when he's available, but the announcements have 3 fields: title, description and body in case we need to show diff info in each